code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm

```diff
@@ -107,11 +106,11 @@
 	additional_items = list(/obj/item/roguegem/diamond)	
 	created_item = /obj/item/clothing/ring/diamond
 
-//Steel rings
+//Silver rings
 
 /datum/anvil_recipe/valuables/silver/emering
 	name = "Gemerald Ring (+1 Gemerald)"
-	req_bar = /obj/item/ingot/steel
+	req_bar = /obj/item/ingot/silver
```

Comment:
Did you know that you could make silver gemerald rings out of steel? not anymore.